### DISCUSSION TITLE
My friends' activity no longer appears in my GitHub dashboard.

### DISCUSSION THREAD

**Discussion by Profitah** [community/community#138492] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I am a GitHub user.
I haven't changed my GitHub settings in years.
But suddenly I realized that when my GitHub friends did something like create a repo
sitory, I couldn't see it.
I also checked the search filter settings on the GitHub main screen, and nothing had changed.

<img width="469" alt="스크린샷 2024-09-13 오전 12 12 20" src="https://github.com/user-attachments/assets/4b2664a2-4fb9-415f-be79-52140f86f75e">

**Comment by Sin-cy** [comment]
Its probably because they have 
"Make profile private and hide activity turned on". 
Its a setting inside your profile settings :)
<img width="638" alt="Screenshot 2567-09-13 at 03 36 56" src="https://github.com/user-attachments/assets/79bd1c90-bec2-45cc-aced-4eb12d48138e">

**Reply by Profitah** [reply]
Is this option a trend? oh my god...😢😢

Thank you for your reply.

**Reply by Nyxox-debug** [reply]
I'm having the same issue, I can't still see there activity

**Reply by Sin-cy** [reply]
Did the answer above not solve your issue ? You are not able to see their activity because they have that setting turned on my guy @Ik-cyber
